src/scss/win/_utilities.scss

```diff
@@ -1,3 +1,11 @@
+.no-outline {
+    &:focus,
+    &:focus:active,
+    &:active {
+        outline: none !important;
```

Comment:
I removed the !important. I had to add more specific selectors, but it still works.